docs/installation/installation-using-docker-windows-10-pro-higher.md

```diff
@@ -24,20 +24,13 @@ cd project-base
 *Note: The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
 
 ### 2. Create docker-compose.yml file
-Create `docker-compose.yml` from template [`docker-compose.yml.dist`](../../project-base/docker/conf/docker-compose.yml.dist).
+Create `docker-compose.yml` from template [`docker-compose-win.yml.dist`](../../project-base/docker/conf/docker-compose-win.yml.dist).
 
 ```
-copy docker\conf\docker-compose.yml.dist docker-compose.yml
+copy docker\conf\docker-compose-win.yml.dist docker-compose.yml
 ```
 
-### 3. Create folder for persisting Postgres data
-This step is necessary to persist database data in cases you need to shutdown you `php-fpm` container. 
-
-```
-mkdir var\postgres-data
-```
-
-### 4. Grant Docker access to your files
+### 3. Grant Docker access to your files
```

Comment:
You've changed 4. to 3., but forgot to change 5. and 6. Why do we keep numbering the headings? Obviously it's not very easy to maintain.

Reply:
Changed